afew/tests/test_dkimvalidityfilter.py

```diff
@@ -0,0 +1,158 @@
+"""Test suite for DKIMValidityFilter.
```

Comment:
I think we need the license template here:
```
# SPDX-License-Identifier: ISC
```

Reply:
Do we really need to do that? Cannot we just have top level LICENCE file, please?

Reply:
Yes sure

Reply:
We don't need the SPDX License Identifier everywhere, and I personally wouldn't add it to new files.

We replaced the verbose license texts in each file with SPDX identifiers - as a compromise, given we can't easily remove other authors' copyright headers/licensing text, but SPDX headers are considered equivalent by most.